index.html

```diff
@@ -824,7 +833,297 @@ <h3>
       </p>
 
     </section>
+    <p class="note">
+      See [[DID-CORE]] for normative requirements associated with the
+      <a href="https://www.w3.org/TR/did-core/#data-model">Data Model</a>.
+    </p>
+
+    <p class="note">
+      See [[DID-CORE]] for normative requirements associated with
+      <a href="https://www.w3.org/TR/did-core/#representations">Representations</a>.
+    </p>
+
+    <p>
+      The DID document data model contains “properties” (such as `id`, `verificationMethod`, and `service`), which are
```

Comment:
Avoid parentheticals, it makes the prose harder to read because it largely distracts from the main point being made. Try commas first, and only when absolutely necessary and as a last resort, use parentheticals.

Reply:
This isn't my content, but I will add it to editorial checklist.